src/main/java/dev/meyi/bn/modules/SuggestionModule.java

```diff
@@ -45,10 +45,10 @@ protected void draw() {
 
     } else {
       Utils.drawCenteredString("Waiting for bazaar data", x + 100,
-          y + (Minecraft.getMinecraft().fontRendererObj.FONT_HEIGHT * 10 + 16) / 2, 0xAAAAAA, 1F);
+          y + (Minecraft.getMinecraft().fontRendererObj.FONT_HEIGHT * 16 + 16) / 2, 0xAAAAAA, 1F);
```

Comment:
default size when waiting should be 10 (or the current flipping list length), not 16.

Reply:
wasn´t intended